tests/helpers.ts

```diff
@@ -6,10 +6,22 @@ import { Project } from 'scenario-tester';
 import { fileURLToPath } from 'url';
 import { expect } from 'vitest';
 
-import { restoreAddon } from './../src/captured-addon/utils/fs-helpers';
+import { captureAddon, restoreAddon } from './../src/captured-addon/utils/fs-helpers';
 
 const __dirname = dirname(fileURLToPath(import.meta.url));
 
+export async function snapshotFrom(addonPath: string) {
```

Comment:
Why alias captureAddon?

Reply:
to simplify scope in tests (to not introduce extra import to deal with)

Reply:
Does it need to be exported? Cal the usage be inlined?

Reply:
oh, my bad, it's not used outside of this file, removed